src/systemcmds/motor_ramp/motor_ramp.cpp

```diff
@@ -75,11 +76,15 @@ enum Mode {
 static bool _thread_should_exit = false;		/**< motor_ramp exit flag */
 static bool _thread_running = false;		/**< motor_ramp status flag */
 static int _motor_ramp_task;				/**< Handle of motor_ramp task / thread */
-static float _ramp_time;
+static float _ramp_time = 1.0f;
 static int _min_pwm;
-static int _max_pwm;
+static int _max_pwm = 2000;
```

Comment:
Please reset these default values inside `motor_ramp_main`, at the very top. Otherwise you get inconsistent results, e.g. with:
```
motor_ramp sine -a 1100 -r 0.5  -b 1800
motor_ramp sine   <- will use 1800 as max pwm, not 2000.
```